### PR TITLE
Kit 1

### PR DESCRIPTION
**Pull request by Newaluigi**
First PR on homepage design

### CODE REVIEW

**Comment by Newaluigi** [comment]
- For the index.html conflict I add the following line for importing some icons <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
    integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
    crossorigin="anonymous" referrerpolicy="no-referrer" />
-For the style.scss conflict I  didn't change anything so I think it isn't updated about the last Tristan's changes

**Review by MatteiT — APPROVED** [review]
It's a massive piece of code for 3 days of work ! An error on my part to let your do this US alone. Let's see what Ben will say about about the css and details but it's seems ok for me ! I'll advice to extend this work on Sprint 2 .

**Review by MatteiT — APPROVED** [review]

**Review by MatteiT — APPROVED** [review]

**Review by MatteiT — DISMISSED** [review]

**Review by MatteiT — DISMISSED** [review]

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -1,8 +1,12 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { HomepageComponent } from './pages/homepage/homepage.component';
 import { RssComponent } from './pages/rss/rss.component';
 
-const routes: Routes = [{ path: 'rss', component: RssComponent }];
+const routes: Routes = [
+  { path: 'home', component: HomepageComponent },
```
Comment (Benja-san) [inline src/app/app-routing.module.ts:7]: if homepage is the default page please add a redirect path '' to home

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/app/pages/homepage/homepage.component.scss [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  height: 200vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  // background-color: $quaternary-color;
```
[the remark this answers is not in the document]
Comment (Newaluigi) [reply src/app/pages/homepage/homepage.component.scss:16]: removed

File: src/app/components/homepage/homeform/homeform.component.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { Component } from '@angular/core';
+// import { UilAt, UilLockAlt } from '@iconscout/unicons
+
+@Component({
+  selector: 'app-homeform',
+  templateUrl: './homeform.component.html',
+  styleUrls: ['./homeform.component.scss'],
+})
+export class HomeformComponent {
+  // UilAt = UilAt;
```
[the remark this answers is not in the document]
Comment (Newaluigi) [reply src/app/components/homepage/homeform/homeform.component.ts:10]: removed

File: src/app/components/homepage/homeform/homeform.component.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { Component } from '@angular/core';
+// import { UilAt, UilLockAlt } from '@iconscout/unicons
```
[the remark this answers is not in the document]
Comment (Newaluigi) [reply src/app/components/homepage/homeform/homeform.component.ts:2]: removed

File: src/app/components/homepage/homeform/homeform.component.scss [hunk 5]
```diff
@@ -0,0 +1,330 @@
...
+  text-align: center;
+  width: 60px;
+  height: 16px;
+  border-radius: 8px;
+  padding: 0;
+  margin: 10px auto;
+  cursor: pointer;
+  background-color: $main-color;
+}
+.checkbox:checked + label:before,
```
[the remark this answers is not in the document]
Comment (Newaluigi) [reply src/app/components/homepage/homeform/homeform.component.scss:107]: With : it works on my browser and some doesn't works with ::

**Review by Benja-san — APPROVED** [review]

**Review by MatteiT — APPROVED** [review]

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (Newaluigi) [reply src/app/app-routing.module.ts]: fixed

**Pull request merged**